src/init.lua

```diff
@@ -8,13 +8,24 @@ local makeThunkMiddleware = require(script.makeThunkMiddleware)
 
 local actions = require(script.types.actions)
 local reducers = require(script.types.reducers)
+local store = require(script.types.store)
+local thunks = require(script.types.thunks)
 
 export type Action<Type = any> = actions.Action<Type>
 export type AnyAction = actions.AnyAction
 export type ActionCreator<Type, Action, Args...> = actions.ActionCreator<Type, Action, Args...>
 
 export type Reducer<State = any, Action = AnyAction> = reducers.Reducer<State, Action>
 
+export type BoundDispatch = store.BoundDispatch
```

Comment:
`BoundDispatch` seems like a leaky abstraction, why is it relevant to external consumers whether or not the dispatch is "bound"?

Reply:
The BoundDispatch types were intended to be used in mapDispatchToProps, but now that I think of it this might be better suited to live in roact-rodux. This issue was that mapDispatchToProps gives consumers a bound dispatch callback, whereas thunks invoke dispatch directly on the store object, so the signature is different.

Reply:
Yeah, that difference makes sense. From what I can see, it's probably reasonable for RoactRodux to define those types itself based on the provided types you're exporting.